pkgs/by-name/re/resources/package.nix

```diff
@@ -48,8 +50,19 @@ stdenv.mkDerivation (finalAttrs: {
     glib
     gtk4
     libadwaita
+    dmidecode
+    util-linux
   ];
```

Comment:
They are only runtime deps so we don't need to add them here.

Reply:
That's why they're in buildInputs - they are needed at run-time.

Reply:
Run time deps doesn't need to be in buildInputs, only build time deps need to be here. We make sure the runtime deps are available via the patches.

Reply:
@amaxine I guess you forgot about this one. Could you remove the two lines please and then I think this one can be merged. :)

Reply:
Nope, just been on a very away from any computers holiday :-) I'll finish it up soon-ish.